Tasks/UsePythonVersionV0/task.json

```diff
@@ -35,6 +35,14 @@
             "defaultValue": "3.x",
             "helpMarkDown": "Version range or exact version of a Python version to use, using semver's version range syntax. [More information](https://go.microsoft.com/fwlink/?LinkID=2006180)"
         },
+        {
+            "name": "allowUnstable",
```

Comment:
Could it make sense to re-check naming for this one? 'allowUnstable' sounds like it just allows to download unstable python release, but generally it uses the different source (specific registry), which could be unobsious.

Reply:
Sorry, see it now - it just does what it meant to be 😃